README.md

```diff
@@ -47,6 +61,36 @@ To move a window, hold your `moveModifiers` down, then click and drag a window.
 
 To resize a window, hold your `resizeModifiers` down, then click and drag a window.
 
+### Fancy zones
+
+Fancy zones module is inspired by [Microsoft's FancyZones](https://docs.microsoft.com/en-gb/windows/powertoys/fancyzones) and aims to provide a similar way to interact with windows in macOS
+
+To configure your fancy zones you have to specify it them in the configuration using the ratios of the screen.
+
+<img alt="Zones example" src="./doc/zones.png" />
+
+The 0,0 point starts in the top-left corner. In order to obtain the result shown in the image you need 9 zones:
+
+```lua
+  zones = {
+    {w='0.3',h='0.3',x='0'  ,y='0'},
+    {w='0.3',h='0.3',x='0.3',y='0'},
+    {w='0.3',h='0.3',x='0.6',y='0'},
+    {w='0.3',h='0.3',x='0'  ,y='0.3'},
+    {w='0.3',h='0.3',x='0.3',y='0.3'},
+    {w='0.3',h='0.3',x='0.6',y='0.3'},
+    {w='0.3',h='0.3',x='0'  ,y='0.6'},
+    {w='0.3',h='0.3',x='0.3',y='0.6'},
+    {w='0.3',h='0.3',x='0.6',y='0.6'},
+  },
+```
+
+Once the zones are configured you can press the modifier key and drag the window to the zones to snip it.
```

Comment:
I don't understand the phrase "snip it" as a reader – can you be more clear and precise here?

Reply:
It is a typo, I was referring to snap as in Windows AeroSnap but I rewrote that part to be clear